ODIN_II/SRC/verilog_flex.l

```diff
@@ -687,7 +690,7 @@ std::string get_complex_define()
 				current_define_body.erase(pos, original_arg.size());
 				current_define_body.insert( pos, replacement_arg );
 				pos += replacement_arg.size();
-				pos = current_define_body.find( replacement_arg, pos );
+				pos = current_define_body.find( original_arg, pos );
```

Comment:
@emacdo12 there was a bug in the defines were arguments were not properly replaced, this PR should get that fixed and allow you to continue the work

Reply:
Ya I'll do that after the headerbug